charts/rasa-x/templates/db-migration-service-deployment.yaml

```diff
@@ -61,6 +61,10 @@ spec:
 {{ toYaml .Values.dbMigrationService.extraVolumeMounts | indent 8 }}
 {{- end }}
         env:
+        {{- if contains "rasa-x-ee" .Values.dbMigrationService.name }}
+        - name: "RASA_TELEMETRY_ENABLED"
```

Comment:
Can we always disable it? We don't track anything in there and I don't we should start tracking in the database migration container.

Reply:
Hmm, good point. I took it from the other part helm chart, but I think you're right and we can just disable it for the DB migration service.